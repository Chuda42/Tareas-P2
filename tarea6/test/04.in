# Implementar sonVecinos
sonVecinos 2 5
sonVecinos 5 2
hacerVecinos 5 2 25
sonVecinos 2 5
sonVecinos 5 2
hacerVecinos 5 8 58
sonVecinos 8 5
sonVecinos 5 8
sonVecinos 8 2
sonVecinos 2 8
hacerVecinos 9 1 91
sonVecinos 1 9
hacerVecinos 2 10 210
sonVecinos 10 2
sonVecinos 1 1
sonVecinos 4 4
sonVecinos 10 10
Fin
