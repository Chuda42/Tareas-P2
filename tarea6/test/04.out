1>#  Implementar sonVecinos.
2>NO son vecinos.
3>NO son vecinos.
4>Nuevos vecinos: 5 2 25.00.
5>Son vecinos.
6>Son vecinos.
7>Nuevos vecinos: 5 8 58.00.
8>Son vecinos.
9>Son vecinos.
10>NO son vecinos.
11>NO son vecinos.
12>Nuevos vecinos: 9 1 91.00.
13>Son vecinos.
14>Nuevos vecinos: 2 10 210.00.
15>Son vecinos.
16>NO son vecinos.
17>NO son vecinos.
18>NO son vecinos.
19>Fin.
